test-app/tests/integration/components/toucan-form/toucan-form-test.gts

```diff
@@ -119,12 +132,30 @@ module('Integration | Component | ToucanForm', function (hooks) {
     assert.dom('[data-checkbox-group-1]').isChecked();
     assert.dom('[data-checkbox-group-2]').isNotChecked();
     assert.dom('[data-checkbox-group-3]').isChecked();
+
+    // File input
+    assert.dom('[data-files] [data-file-name]').hasText('file.txt');
   });
 
   test('it triggers validation and shows error messages in the Toucan Core components', async function (assert) {
-    assert.expect(13);
+    assert.expect(19);
+
+    const handleSubmit = ({ files, ...data }: TestData) => {
+      // Checking deep equality on files can get really tricky due
+      // to properties on the File object prototype like `arrayBuffer, `lastModifiedDate`,
+      // and `lastModified`.  Due to that, we will assert `files` separately from the other
+      // submitted values.
+      assert.ok(files, 'Expected files to be included');
+      assert.strictEqual(
+        files?.length,
+        1,
+        'Expected only a single file to be added'
+      );
+      assert.strictEqual(files?.[0]?.name, 'file.txt');
+      assert.strictEqual(files?.[0]?.type, 'text/plain');
+      assert.strictEqual(files?.[0]?.size, 19);
```

Comment:
Super open to any suggestions here, but figured this is probably safe enough.  I just didn't want to get into the business of comparing `lastModified` and `lastModifiedDate`.

Reply:
Where is that `19` coming from? Is there a real file that we are using here, that has this size? Maybe add a comment to explain, as it was not obvious to me?

Reply:
Good idea, I'll add a comment. Is indeed based on our test file we create. Alternatively we could NOT assert the size if we want, as the other assertions are probably fine enough?